### PR TITLE
Privacy page

### PR DESCRIPTION
**Pull request by CatalinVoineag**
## Changes proposed in this pull request

Privacy page
Footer helper

## Guidance to review

Review content

## Screenshots

Screenshot from 2024-03-21 14-33-32

### CODE REVIEW

**Comment by CatalinVoineag** [comment]
[Pull in privacy notice](https://trello.com/c/0mV2qYzt/214-pull-in-privacy-notice)

File: app/views/claims/pages/privacy.html.erb [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<% content_for :page_title, t(".page_title") %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t(".page_title") %></h1>
+
+      <h2 class="govuk-heading-m">Who we are</h2>
+      <p class="govuk-body">This work is being carried out by Department for Education (DfE) Digital, which is a part of DfE. For the purpose of data protection legislation, the DfE is the data controller for the personal data processed as part of Claim funding for mentor training.</p>
```
Comment (Nitemaeric) [inline app/views/claims/pages/privacy.html.erb:9]: Hmm, should this be in a locale file?

```yaml
privacy:
  page_title: Title
  body:
    - heading: Who we are
      body: This work ...
    - Why our use
```

This way, we could potentially localise it. We'll likely not need to, but the effort of adding this to a locale file _now_ vs converting it into a locale + adding a new locale down the line seems like a tradeoff we can invest in now.

Unlikely, but Welsh _might_ be a language we need to support?

Comment (CatalinVoineag) [reply app/views/claims/pages/privacy.html.erb:9]: We had a similar conversation here https://github.com/DFE-Digital/itt-mentor-services/pull/380#discussion_r1530109450

And I think @olioldfield confirmed we won't be serving Welsh schools?

Comment (Nitemaeric) [reply app/views/claims/pages/privacy.html.erb:9]: 👍 TIL localised views. `[page].[locale].html.erb` for static pages sounds good to me :)

**Review by Nitemaeric — APPROVED** [review]

**Review by Kizr — APPROVED** [review]

**Pull request merged**